File: javascript/ex018.js
console.clear();

const angulo =  45;
const seno = Math.sin(angulo * (Math.PI / 180));
const cosseno = Math.cos(angulo * (Math.PI / 180));
const tangente = Math.tan(angulo * (Math.PI / 180));

console.log(`
o Angulo de ${angulo} tem o seno de: ${seno.toFixed(2)},
Cosseno: ${cosseno.toFixed(2)},
Tangente: ${tangente.toFixed(2)}
`);

File: python3/ex018.py
import math

angulo = float(input("Digite o angulo que voce quer: "))
seno = math.sin(math.radians(angulo))
cosseno = math.cos(math.radians(angulo))
tangente = math.tan(math.radians(angulo))

print(f"""
o Angulo de {angulo} tem o seno de: {seno:.2f},
Cosseno: {cosseno:.2f},
Tangente: {tangente:.2f}
""")

